src/test/java/pagefactory/HomePage.java

```diff
@@ -0,0 +1,37 @@
+package pagefactory;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage{
+
+    @FindBy(css = "img['class=avatar']")
+    WebElement avatarIcon;
+    @FindBy(css = ".playlist:nth-child(3)")
+    WebElement firstPlaylist;
+    @FindBy(css ="[name='name']")
+    WebElement playlistNameField;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```